static/js/factories/learning_resources.js

```diff
@@ -163,7 +163,8 @@ export const makeUserList = (): UserList => ({
   list_type:     "userlist",
   profile_img:   casual.url,
   profile_name:  casual.name,
-  privacy_level: casual.random_element(["public", "private"])
+  privacy_level: casual.random_element(["public", "private"]),
+  author:        casual.number
```

Comment:
I think here for `author` you need to use `casual.integer(low, high)`, where `low` and `high` are integer values for a range from which the function should select an integer (e.g. 1 and 1000 or something)

Reply:
just realized we are using `casual.number` in a number of places - but if you check in the repl `casual.number` isn't a function that `casual` provides

Reply:
I replaced all the `casual.number`s with `casual.integer`s